espresso-server/app/src/androidTest/java/io/appium/espressoserver/lib/http/Router.java

```diff
@@ -224,6 +224,9 @@
         routeMap.addRoute(new RouteDefinition(Method.POST, "/session/:sessionId/appium/execute_mobile/backdoor", new MobileBackdoor(), MobileBackdoorParams.class));
         routeMap.addRoute(new RouteDefinition(Method.POST, "/session/:sessionId/appium/execute_mobile/:elementId/flash", new MobileViewFlash(), ViewFlashParams.class));
 
+        // Compatibility for https://github.com/appium/appium-base-driver/blob/master/lib/jsonwp-proxy/protocol-converter.js
```

Comment:
actually, this route is the correct one for the JSONWP protocol. Having a comment the W3C one is an alias to this one would be enough: https://github.com/appium/appium-uiautomator2-server/blob/20de6220d4d9b888892b8e148a1e16b6aa8b874d/app/src/main/java/io/appium/uiautomator2/server/AppiumServlet.java#L157